tests/validation/rules/checked.spec.ts

```diff
@@ -3,9 +3,16 @@ import { checked } from 'sefirot/validation/rules'
 describe('validation/rules/checked', () => {
   test('it validates whether the value is true', () => {
     const rule = checked()
+
     expect(rule.$validator(undefined, null, null)).toBe(true)
+    expect(rule.$validator(null, null, null)).toBe(true)
     expect(rule.$validator(true, null, null)).toBe(true)
+    expect(rule.$validator([], null, null)).toBe(true)
```

Comment:
Note that, with `_required` validator, empty array is treated as valid so that every rule with `optional: true` returns `true for array.

I guess passing array to the rules that don't expect array would not happen in real usecases, but anyway I tested all the possible cases.

Reply:
Gotcha. I too think that's OK 👍